org.civicrm.hremergency/api/v3/Contact/Deleteemergencycontact.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Contact.DeleteEmergencyContact API specification
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ */
+function _civicrm_api3_contact_deleteemergencycontact_spec(&$spec) {
+  $spec['id']['api.required'] = 1;
+}
+
+/**
+ * Contact.DeleteEmergencyContact API
+ *
+ * @param array $params
+ * @return array API result descriptor
+ */
+function civicrm_api3_contact_deleteemergencycontact($params) {
+  civicrm_api3_verify_mandatory($params, NULL, ['id']);
```

Comment:
I can add this, but just wondering should this be added by default for all entities + actions? If so it might make more sense to put it in hrcore and apply to all (unless we need a whitelist for unauthenticated users)

Reply:
I can't be a 100% sure right now, but I don't we need, or will need, such whitelist. I'm also not sure about giving this permission for all entities and actions, but I think we might have some custom APIs that should only be accessible to users with a more restrictive permission. For now, let's keep it as is. At the moment, I think it makes more sense for the extension to define the permission for its own APIs.